x/simulation/account.go

```diff
@@ -32,21 +32,29 @@ func RandomAcc(r *rand.Rand, accs []Account) Account {
 }
 
 // RandomAccounts generates n random accounts
-func RandomAccounts(r *rand.Rand, n int) []Account {
+func RandomAccounts(r *rand.Rand, n int, blackList map[string]bool) []Account {
```

Comment:
I think the check against `blackList` should be removed from here as well. Shouldn't do this in `RandomAcc` because `RandomAcc` gets called in all the different modules in `x/`. We don't want to have that `blackList` passed in to each module that wants to use simulation test.

I think we should just remove `blackList` checking here and do one final check in `SimulateFromSeed`